api/src/team/application/api/certification-center.api.js

```diff
@@ -0,0 +1,13 @@
+import { usecases } from '../../domain/usecases/index.js';
+
+/**
+ * @function
+
+ * @param {number} certificationCenterId
+ * @param {date} archiveDate
+ * @param {number} archivedBy
+ */
+
+export async function archiveCertificationCenterData({ certificationCenterId, archiveDate, archivedBy }) {
```

Comment:
Enlever les lignes vides dans les JSDoc et entre une JSDoc et le code auquel elle se rapporte : 

```suggestion
/**
 * @function
 * @param {number} certificationCenterId
 * @param {date} archiveDate
 * @param {number} archivedBy
 */
export async function archiveCertificationCenterData({ certificationCenterId, archiveDate, archivedBy }) {
```